saray/message-page/styles.css

```diff
@@ -0,0 +1,77 @@
+.view-port {
+    align-items: center;
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+}
+
+.font-lato {
+    font-family: 'Lato', 'Arial', sans-serif;
+}
+
+.container {
+    display: flex;
+    flex-direction: row;
+    font-family: 'Lato', 'Arial', sans-serif;
```

Comment:
You don't need put this property to container instead you already have created the class ```font-lato```. Put this class to the HTML elements.